1-getting-started/lessons/3-sensors-and-actuators/translations/pi-actuator.kr.md

```diff
@@ -0,0 +1,116 @@
+# 야간 조명 만들기 - 라즈베리 파이
```

Comment:
Didn't we decide to write Raspberry Pi in English?

Reply:
Oh... I miss that one... Thanks!